observing/parsesdf.py

```diff
@@ -206,9 +224,11 @@ def power_beam_obs(obs_list, session, mode='buffer'):
         pointing_buffer = 0.1
         recording_buffer = 0.1
 
-    if session.do_cal:
-        dt = (configure_buffer + cal_buffer + controller_buffer + pointing_buffer + recording_buffer)/3600/24
-    elif not session.do_cal:
+    if session.cal_directory is not None and session.do_cal == True:
```

Comment:
I think some other edits are required. I'd originally allowed calibrations for for cal_dir = None, it would just instead use the calibration directory that is defined in the config file

Reply:
Good to know your intentions here. I will add a validation check on the cal_directory file.
The cal_directory argument makes sense as a way to overload the value set in the config file. But if it is not set, I think it reasonable to expect the mnc-python code to use the default value in the config file. I'll push these changes to the PR and you let me know what you think.